Standards/CCSMB-3.md

```diff
@@ -0,0 +1,24 @@
+# CCSMB-3: Directory
+
+*Author: Oliver Cooper (oeed)*
+
+*Last updated: 2017-01-27*
+
+*Version: v1.0.0*
+
+## Quick information
+
+| Information |                           |
+| ----------- | ------------------------- |
+| Version     | 1.0.0                     |
+| Type        | Folder/directory          |
+| MIME        | `application/directory`   |
+| Extensions  | *See 'Extensions'*        |
+
+## Extensions
+
+In short: *anything*.
```

Comment:
Making a standard and then saying "this can be anything at all" isn't really standardization.